tests/parsers/agilent_gen5_image/agilent_gen5_image_structure_test.py

```diff
@@ -4,112 +4,14 @@
     TransmittedLightSetting,
 )
 from allotropy.parsers.agilent_gen5_image.agilent_gen5_image_structure import (
-    HeaderData,
     InstrumentSettings,
-    LayoutData,
     ReadData,
     ReadSection,
 )
 from allotropy.parsers.agilent_gen5_image.constants import DetectionType
 from allotropy.parsers.lines_reader import LinesReader
 
 
-@pytest.mark.short
```

Comment:
Note to reviewer: these tests are deleted because gen_5_image structure re-uses identical functions from gen_5 which are already tested